src/peerbook/libp2p_peer.proto

```diff
@@ -14,6 +14,12 @@ message peer {
     repeated bytes connected = 3;
     nat_type nat_type = 4;
     int64 timestamp = 5;
+    repeated association associations = 6;
+}
+
+message association {
+    bytes address = 1;
```

Comment:
It would be nice to support more than one type of association. Eg. if we used libolm for the 'encrypted channels', we could store curve25519 keys in here to allow the bootstrap of the encrypted session.